### PR TITLE
Offer add intent testing

### PR DESCRIPTION
**Pull request by offerakrabi**
no need to test the tests...

this will let us to add utterances and expected outputs for each intent.

before we merge this we should think if there is something we can add or if there's a better way to do this

### CODE REVIEW

**Comment by erezbi** [comment]
@offerakrabi you do need to test that the tests are passing currently :)

**Comment by offerakrabi** [comment]
@erezbi 
I checked manually in the skillboilerplate that everything is working.. I don't think we have any tests in our test suit testing the unit tests.

File: test/actions.js [hunk 1]
```diff
@@ -7,55 +7,128 @@
...
+const skillManifest = require(process.env.skillSDKResDir + '/assets/manifest');
 // Bring the actions
 require('..');
 
 describe('actions', function() {
 
-  describe('converse', function() {
+    describe('converse', function() {
+        for (let test of utteranceFile.tests) {
+            it('intent ' + test.intent + ' with utterance: ' +  '\'' + test.utterance + '\'' + ', should return: '
```
Comment (erezbi) [inline test/actions.js]: Please add 'description' field to test definition and print it here.

File: test/actions.js [hunk 2]
```diff
@@ -7,55 +7,128 @@
...
-    it('intent hello-world us-EN', function(done) {
+                request(server)
+                    .post('/v1/api/converse')
+                    .send({
+                        id: '001',
+                        text: test.utterance,
+                        retext: test.utterance,
+                        version: '1.0',
+                        language: 'en-US',
+                        attributes: {
```
Comment (erezbi) [inline test/actions.js]: Add entities.

File: test/actions.js [hunk 3]
```diff
@@ -7,55 +7,128 @@
...
+                        }
+                    })
+                    .set('Accept', 'application/json')
+                    .expect('Content-Type', /json/)
+                    .expect(200)
+                    .end(function(err, res) {
+                        should.not.exist(err);
+                        res.body.should.have.property('speech');
+                        res.body.speech.text.should.match(test.expected_response);
+                        res.body.should.have.property('deleteSkillSession').which.be.exactly(true);
```
Comment (erezbi) [inline test/actions.js]: remove this line

File: test/actions.js [hunk 4]
```diff
@@ -7,55 +7,128 @@
...
+        it('wcs', function (done) {
+            request(server)
+                .get('/v1/api/nlu?type=wcs')
+                .set('Accept', 'application/json')
+                .expect('Content-Type', /json/)
+                .expect(200)
+                .end(function (err, res) {
+                    should.not.exist(err);
+                    res.body.should.have.property('workspace');
+                    res.body.should.have.property('credentials');
```
Comment (erezbi) [inline test/actions.js:95]: Should have property 'url'.

File: test/actions.js [hunk 5]
```diff
@@ -7,55 +7,128 @@
...
+                    should.notEqual(res.body.credentials.username, 'your wcs username', 'Username is not set.');
+                    should.notEqual(res.body.credentials.password, 'your wcs password', 'Password is not set.');
+                    done();
```
Comment (erezbi) [inline test/actions.js]: add the same check for URL.

**Review by erezbi — APPROVED** [review]

**Pull request merged**